rules/aip0158/response_plural_first_field.go

```diff
@@ -30,7 +31,16 @@ var responsePluralFirstField = &lint.MessageRule{
 		// Throw a linter warning if, the first field in the message is not named
 		// according to plural(message_name.to_snake().split('_')[1:-1]).
 		firstField := m.GetFields()[0]
-		want := utils.ToPlural(firstField.GetName())
+
+		// If the field is a resource, use the `plural` annotation to decide the
+		// appropriate plural field name.
+		var want string
+		if res := utils.GetResource(firstField.GetMessageType()); res != nil && res.GetPlural() != "" {
```

Comment:
nit: make a util function similar to GetResourceSingular? https://github.com/googleapis/api-linter/blob/ececf6c459201eccbaf506e57d0f735c62ba7aa0/rules/internal/utils/resource.go#L25.

Reply:
Good call! Done.